core/chains/evm/gas/rollups/l1_oracle_test.go

```diff
@@ -19,16 +19,24 @@ import (
 
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/chaintype"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/toml"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas/rollups/mocks"
 )
 
+const (
+	OPGasOracleAddress     = "0x420000000000000000000000000000000000000F"
```

Comment:
If it's used only in tests then probably you don't need these const values. You can replace them with randomized addresses `utils.RandomAddress()`